swap/src/protocol/alice/swap.rs

```diff
@@ -81,10 +64,31 @@ pub fn is_encsig_learned(state: &AliceState) -> bool {
     )
 }
 
+pub async fn run(swap: alice::Swap) -> Result<AliceState> {
+    run_until(swap, is_complete).await
+}
+
+pub async fn run_until(
```

Comment:
```suggestion
pub async fn run_swap_until(
```

Reply:
I think `run` and `run_until` are fine. The swap is the first parameter. That implies that we are running the swap.